admin/src/components/DeletedUserFormular.vue

```diff
@@ -4,15 +4,21 @@
       {{ $t('removeNotSelf') }}
     </div>
     <div v-else class="mt-5">
-      <b-form-checkbox switch size="lg" v-model="checked">
-        <div>{{ item.deletedAt ? $t('undelete_user') : $t('delete_user') }}</div>
-      </b-form-checkbox>
-
       <div class="mt-3 mb-5">
-        <b-button v-if="checked && item.deletedAt === null" variant="danger" @click="deleteUser">
+        <b-button
+          v-if="item.deletedAt === null"
+          variant="danger"
+          v-b-modal.delete-user-modal
+          @click="showDeleteModal()"
+        >
           {{ $t('delete_user') }}
         </b-button>
-        <b-button v-if="checked && item.deletedAt !== null" variant="success" @click="unDeleteUser">
+        <b-button
+          v-if="item.deletedAt !== null"
```

Comment:
You check the opposite in the previous block
```suggestion
          v-else
```